src/main/java/org/spongepowered/api/command/spec/CommandSpec.java

```diff
@@ -354,6 +354,15 @@ public boolean testPermission(CommandSource source) {
     }
 
     /**
+     * Gets the extended description used with this command if any is present
+     *
+     * @return the extended description.
+     */
+    public Optional<Text> getExtendedDescription(CommandSource source) {
```

Comment:
Technically, this doesn't even need to require the `source` param, though keeping it for consistencies sake might be a good idea. Thoughts?